cert_manager/tests/common.py

```diff
@@ -1,7 +1,7 @@
 # (C) Datadog, Inc. 2019
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
-from datadog_checks.stubs import aggregator
+from datadog_checks.base.stubs import aggregator
```

Comment:
```suggestion
```

Reply:
`aggregator` is used in this file

Reply:
https://datadoghq.dev/integrations-core/ddev/plugins/#fixtures